### DISCUSSION TITLE
How to resolve many-to-many relation with "through" option

### DISCUSSION THREAD

**Discussion by rajkrishnanind** [graphql-python/graphene-django#1175] [Q&A]
I have raised a question in stack overflow. Here is the link
[https://stackoverflow.com/questions/53013384/how-to-resolve-graphene-django-node-field-with-many-to-many-relationship](url)

I was expecting it to be auto resolved as the relations are defined in models. Looking forward to get some help.

**Comment by dopeboy** [comment]
@rajkrishnanind, looks like you find a solution per your SO thread. Closing this down.
